gnatsd.go

```diff
@@ -1,9 +1,9 @@
 package test
 
 import (
-	"github.com/fsouza/go-dockerclient"
-	gnatsd "github.com/nats-io/gnatsd/server"
```

Comment:
gnatsd is retired and replaced by nats-server/v2?

Reply:
I guess it's just renamed, not retired. You'll be redirected to https://github.com/nats-io/nats-server when you access https://github.com/nats-io/gnatsd.